### PR TITLE
- fixes #8170: adds a mention for IPs in use for change notifications

### LINKED ISSUE #8170: Link to the IP ranges used by change notifications
I spent many hours trying to track down the IP ranges used by Change Notifactions as we didn't want to open every Azure IP range as inbound connections.  After much heartburn we stumbled upon the correct IP addreses in this article...

https://docs.microsoft.com/en-us/office365/enterprise/additional-office365-ip-addresses-and-urls

To save others the same trouble we had it would be ideal to include a link to this information on this page.  

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bc1ec710-fff6-0690-7eb3-16b9773145ef
* Version Independent ID: c4a14ee1-f979-8409-4481-45272b35d965
* Content: [Set up notifications for changes in user data - Microsoft Graph](https://docs.microsoft.com/en-us/graph/webhooks#feedback)
* Content Source: [concepts/webhooks.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/concepts/webhooks.md)
* Product: **non-product-specific**
* Technology: **microsoft-graph**
* GitHub Login: @baywet
* Microsoft Alias: **MSGraphDocsVteam**

### PR DESCRIPTION
**Pull request by baywet**

### CODE REVIEW

File: concepts/webhooks.md [hunk 1]
```diff
@@ -274,6 +274,12 @@ The following code samples are available on GitHub.
 - [Microsoft Graph Webhooks Sample for ASP.NET Core](https://github.com/microsoftgraph/aspnetcore-webhooks-sample)
 - [Microsoft Graph Webhooks Sample for Java Spring](https://github.com/microsoftgraph/java-spring-webhooks-sample)
 
+## Firewall configuration
+
+You optionnally can configure the firewall protecting your notification URL to allow inbound connections only from Microsoft Graph. This allows you to reduce your exposure further to rogue actors sending invalid notifications to your notification URL so they can trigger the custom logic you implemented. For a complete list of IP addresses used by Microsoft Graph to deliver change notifications please refer to [additional endpoints for Office 365](https://docs.microsoft.com/office365/enterprise/additional-office365-ip-addresses-and-urls) under the "Microsoft Graph Change Notifications" entry.
```
Comment (davidmu1) [inline concepts/webhooks.md:279]: Suggest this change: You can optionally configure the firewall that protects your notification URL to allow inbound connections only from Microsoft Graph. This allows you to reduce further exposure to invalid notifications that are sent to your notification URL. These invalid notifications can be trying to trigger the custom logic that you implemented. For a complete list of IP addresses used by Microsoft Graph to deliver change notifications, see [additional endpoints for Office 365](https://docs.microsoft.com/office365/enterprise/additional-office365-ip-addresses-and-urls) under "Microsoft Graph Change Notifications".

File: concepts/webhooks.md [hunk 2]
```diff
@@ -274,6 +274,12 @@ The following code samples are available on GitHub.
...
+
+> **Note:** The IP addresses listed used to deliver change notifications can be updated at any time without notice.
```
Comment (davidmu1) [inline concepts/webhooks.md:281]: Suggest this change: The listed IP addresses that are used to deliver change notifications can be updated at any time without notice.

Comment (baywet) [reply concepts/webhooks.md:281]:
```suggestion
> **Note:** The listed IP addresses that are used to deliver change notifications can be updated at any time without notice.
```

File: concepts/webhooks.md (hunk 1 above)
Comment (baywet) [reply concepts/webhooks.md:279]:
```suggestion
You can optionally configure the firewall that protects your notification URL to allow inbound connections only from Microsoft Graph. This allows you to reduce further exposure to invalid notifications that are sent to your notification URL. These invalid notifications can be trying to trigger the custom logic that you implemented. For a complete list of IP addresses used by Microsoft Graph to deliver change notifications, see [additional endpoints for Office 365](https://docs.microsoft.com/office365/enterprise/additional-office365-ip-addresses-and-urls) under "Microsoft Graph Change Notifications".
```

**Pull request merged**